src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -66,9 +63,9 @@ void JwtAuthenticator::Verify(HeaderMap &headers,
   // http://www.w3.org/TR/cors/#cross-origin-request-with-preflight-0, CORS
   // pre-flight requests shouldn't include user credentials.
   if (headers_->Method() &&
-      LowerCaseString(kOptionsHttpMethod) ==
+      LowerCaseString(Http::Headers::get().MethodValues.Options) ==
```

Comment:
Why do we need to lower case the method?   I thought http methods are pre-defined, they are always upper case.

Reply:
Done: lowercase method has been removed.